src/Avalonia.Themes.Fluent/Controls/SplitButton.xaml

```diff
@@ -0,0 +1,205 @@
+            </Grid.ColumnDefinitions>
+
+            <Button x:Name="PART_PrimaryButton"
+                    Grid.Column="0"
+                    MinWidth="{DynamicResource SplitButtonPrimaryButtonSize}"
+                    Foreground="{TemplateBinding Foreground}"
+                    Background="{TemplateBinding Background}"
+                    BorderThickness="{TemplateBinding BorderThickness}"
+                    BorderBrush="{TemplateBinding BorderBrush}"
+                    Content="{TemplateBinding Content}"
+                    ContentTemplate="{TemplateBinding ContentTemplate}"
+                    Command="{TemplateBinding Command}"
+                    CommandParameter="{TemplateBinding CommandParameter}"
+                    FontFamily="{TemplateBinding FontFamily}"
+                    FontSize="{TemplateBinding FontSize}"
+                    FontWeight="{TemplateBinding FontWeight}"
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Stretch"
+                    HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+                    VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+                    Padding="{TemplateBinding Padding}"
+                    Focusable="False"
+                    KeyboardNavigation.IsTabStop="False" />
+
+            <Button x:Name="PART_SecondaryButton"
+                    Grid.Column="1"
+                    MinWidth="{DynamicResource SplitButtonSecondaryButtonSize}"
+                    BorderBrush="{TemplateBinding BorderBrush}"
+                    Foreground="{TemplateBinding Foreground}"
+                    Background="{TemplateBinding Background}"
+                    BorderThickness="{TemplateBinding BorderThickness}"
+                    HorizontalContentAlignment="Right"
+                    VerticalContentAlignment="Center"
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Stretch"
+                    Focusable="False"
+                    KeyboardNavigation.IsTabStop="False" />
+          </Grid>
+        </ControlTemplate>
+      </Setter.Value>
+    </Setter>
+  </Style>
+
+  <!-- Default overridable styles -->
+  <Style Selector="SplitButton /template/ Button#PART_PrimaryButton">
+    <Setter Property="CornerRadius" Value="{TemplateBinding CornerRadius, Converter={StaticResource LeftCornerRadiusFilterConverter}}" />
+  </Style>
+  <Style Selector="SplitButton /template/ Button#PART_SecondaryButton">
+    <Setter Property="CornerRadius" Value="{TemplateBinding CornerRadius, Converter={StaticResource RightCornerRadiusFilterConverter}}" />
+    <Setter Property="Padding" Value="9,0" />
+    <Setter Property="Content">
+      <Template>
+        <PathIcon Height="12"
+                  Width="12"
+                  Data="{StaticResource SplitButtonSecondaryButtonIconGeometry}" />
+      </Template>
+    </Setter>
+  </Style>
```

Comment:
I'm not sure why you felt the need to pull this out. Take a look at how ComboBox does it. You are unnecessarily requiring a selector to be matched.

If you want the ability for app developers to swap out the DropDownGlyph that's a much larger change and should have a shared resource somewhere for all controls.

Reply:
Yeah, it's not necessary. But it will allow to swap not only glyph, but whole button content. Which sounds common for secondary button.

Reply:
This is standardized in all theme controls: ComboBox, SplitButton future DropDownButton, etc. So app developers really shouldn't be switching it out. Additionally the established precedent from ComboBox in Avalonia doesn't do this. And the established precedent from UWP/WInUI doesn't do this. I've not heard of any complaints not being able to change the drop down glyph in UWP/WinUI. So I have to think your argument doesn't hold up in the real world and this is unnecessary complexity.

If you really wanted it done, it would make sense to make a shared resource for all controls in another PR. Anyway, this type of change is a good example of a change too far. Sure it can be done, but it's not needed in all past precedent and certainly wasn't needed to slip in here. Not all ideas need to drive code changes.

Reply:
I am leaving the SecondaryButton style and content separate as you have it. I still think it is non-ideal and unnecessary considering past precedent. However, it's not worth my time to argue that point here and I've created a new Issue:  https://github.com/AvaloniaUI/Avalonia/issues/7452

Whatever is decided in that other issue should be applied for both ComboBox and SplitButton.

Please mark as resolved if you have no additional comments.